os/kernel/binary_manager/binary_manager_load.c

```diff
@@ -255,10 +255,23 @@ static int binary_manager_load_all(void)
 	load_cnt = 0;
 	bin_count = binary_manager_get_ucount();
 
+	/* First, Load Real-time binaries */
```

Comment:
The following is the same operation twice.
How about separating the functions or following below?
'''

Reply:
Would you please double-check that the intention is correct?

Reply:
We want to load REALTIME first and NONREALTIME later.
Because of this, she made separated loop.
I think we can make function with type param, but we cannot avoid the duplication.

Reply:
i understood.
If the logic is duplicated, it is better to separate it into functions.